lib/absinthe/schema/notation.ex

```diff
@@ -212,7 +212,12 @@ defmodule Absinthe.Schema.Notation do
 
     __CALLER__
     |> recordable!(:object, @placement[:object])
-    |> record!(Schema.ObjectTypeDefinition, identifier, attrs, block)
+    |> record!(
+      Schema.ObjectTypeDefinition,
+      identifier,
+      attrs |> Keyword.update(:description, nil, &wrap_in_unquote/1),
```

Comment:
see function `wrap_in_unquote` for a description.

this `Keyword.update(:description, nil, &wrap_in_unquote/1)` is the fix pasted in all the places necessary. basically is the only change to `notation.ex`